src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIPacket.cs

```diff
@@ -75,7 +75,7 @@ public SNIPacket(SNIHandle owner,int id)
         {
             if (_data != null)
             {
-                Debug.Fail($@"finalizer called for unreleased SNIPacket, tag: {_traceTag}");
+                //Debug.Fail($@"finalizer called for unreleased SNIPacket, tag: {_traceTag}");
```

Comment:
I think my PR https://github.com/dotnet/SqlClient/pull/897 takes care of it, and adds a trace here, you can let this be.

Reply:
Ok, done. Having to take account of all open PR's is a lot of cognitive load. 
I could really do with that sni tracing Pr merging because it's blocking a lot of things.